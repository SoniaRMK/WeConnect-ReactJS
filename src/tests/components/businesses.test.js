import {shallow} from 'enzyme';
import React from 'react';
import BusinessesList from '../../components/business/businesses';

describe('BusinessesList component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<BusinessesList/>)
        expect(wrapper).toMatchSnapshot();
    })
})
