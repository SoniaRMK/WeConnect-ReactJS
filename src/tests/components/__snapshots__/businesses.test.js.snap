// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BusinessesList component should match snapshot 1`] = `
<Route
  render={[Function]}
/>
`;
